.gitignore

```diff
@@ -160,4 +160,5 @@ dotCMS/src/main/webapp/dotcms-binary-field-builder
 dotCMS/src/main/webapp/dotcms-block-editor
 dotCMS/src/main/webapp/WEB-INF/classes
 # allow build to cache files that are not removed on clean e.g. spotless index
-.cache
\ No newline at end of file
+.cache
+!/examples/nextjs/.npmrc
```

Comment:
What this do?

Reply:
This file has the preferred URL of the npm registry (in this case local verdaddacio)